pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -331,6 +331,11 @@ func (r *ReconcileDeadmansSnitchIntegration) createSnitch(dmsi *deadmanssnitchv1
 			snitch = snitches[0]
 		} else {
 			newSnitch := dmsclient.NewSnitch(snitchName, dmsi.Spec.Tags, "15_minute", "basic")
+			notes := fmt.Sprintf(`
+			cluster_id: %s
+			runbook: https://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md
+			`, cd.Labels["api.openshift.com/id"])
```

Comment:
Just to double check – this is the external cluster id, not the internal (hive) id?

Reply:
nice catch @mmazur !, I've converted it to _externalID_